fingerprint/infra/image-distortion-config/src/main/java/com/simprints/fingerprint/infra/imagedistortionconfig/remote/ImageDistortionConfigRemoteRepo.kt

```diff
@@ -67,6 +67,7 @@ internal class ImageDistortionConfigRemoteRepo @Inject constructor(
         private const val PROJECTS_FOLDER = "projects"
         private const val UN20_MODULES_FOLDER = "un20modules"
         private const val FILE_NAME = "calibration.dat"
-        private const val TAG = "distortionConfigRepo"
+
+        private const val TAG = "DISTORTION_FILE_UPLOAD"
```

Comment:
Shouldn't this be in `CrashReportTag`?

Reply:
I thought about it and decided to leave the tag local to the file if it is only used in a that single file.